### PR TITLE
Issue #25032 fixed

### PR DESCRIPTION
**Pull request by ghost**
### Description (*)

Display some error "We can't update your Wish List right now." at wish list
### Fixed Issues (if relevant)

1. magento/magento2#25032: Display some error "We can't update your Wish List right now." at wish list

### Manual testing scenarios (*)

Scenario 1
1. Front end > Add product to wish list
2. Go to my wish list
3. Edit the product from wish list
4. From detail page leave qty blank and click on Update wish list

Scenario 2
1. Front end > Add product to wish list
2. Go to my wish list
3. Edit the product from wish list
4. From detail page type a negative quantity and click on Update wish list

### Contribution checklist (*)
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds are green)

### CODE REVIEW

**Comment by ihor-sviziev** [comment]
Hi @sidolov or @sivaschenko ,
Could you assist @vishesh-webkul why `Partner: Webkul` wasn't added automatically?

**Comment by sidolov** [comment]
Hi @ihor-sviziev , @vishesh-webkul the label wasn't added automatically because Webkul does not participate in the Partners Contribution Program.

**Comment by rodrigowebjump** [comment]
@magento give me test instance

**Comment by magento-engcom-team** [comment]
Hi @rodrigowebjump. Thank you for your request. I'm working on Magento instance for you

**Comment by magento-engcom-team** [comment]
Hi @rodrigowebjump, here is your [new Magento instance](https://pr-25062.instances.magento-community.engineering).
Admin access: [https://pr-25062.instances.magento-community.engineering/admin](https://pr-25062.instances.magento-community.engineering/admin)
Login: `admin` Password: `123123q`

**Comment by rodrigowebjump** [comment]
@magento give me 2.3-develop instance

**Comment by magento-engcom-team** [comment]
Hi @rodrigowebjump. Thank you for your request. I'm working on Magento 2.3-develop instance for you

**Comment by magento-engcom-team** [comment]
Hi @rodrigowebjump, here is your [Magento instance](https://i-25062-2-3-develop.instances.magento-community.engineering).
Admin access: [https://i-25062-2-3-develop.instances.magento-community.engineering/admin](https://i-25062-2-3-develop.instances.magento-community.engineering/admin)
Login: `admin` Password: `123123q`
Instance will be terminated in up to 3 hours.

**Review by rodrigowebjump — CHANGES_REQUESTED** [review]
Hi @vishesh-webkul 

Your change does not validate if the inputed information is correct.
Please review your change. Try to follow the add to cart validation before submitting.

Thanks

File: app/code/Magento/Wishlist/view/frontend/web/js/add-to-wishlist.js [hunk 1]
```diff
@@ -63,6 +63,15 @@ define([
                 isFileUploaded = false,
                 self = this;
 
+            $('[data-action="add-to-wishlist"]').on('click', function () {
+                var validator = $('#product_addtocart_form').validate();
```
Comment (VladimirZaets) [inline app/code/Magento/Wishlist/view/frontend/web/js/add-to-wishlist.js:67]: Please define the selector in the configuration block. 
Also, all variable declaration should be on the top of the function.

**Comment by VladimirZaets** [comment]
@vishesh-webkul , I am closing this PR now due to inactivity.
Please reopen and update if you wish to continue.
Thank you for collaboration

**Pull request closed without merging**